buildSrc/src/main/java/com/uber/okbuck/core/model/base/Scope.java

```diff
@@ -442,12 +442,7 @@ public Scope build() {
               key,
               t ->
                   new Scope(
-                      project,
-                      useful,
-                      sourceDirs,
-                      javaResourceDirs,
-                      compilerOptions,
-                      depCache));
+                      project, useful, sourceDirs, javaResourceDirs, compilerOptions, depCache));
```

Comment:
why does the formatting change, some inconsistency b/w ide formatting and command line gjf?

Reply:
possibly. I ran this with gjf